README.md

```diff
@@ -21,4 +21,12 @@ dotnet sonarscanner begin /k:"vh-bookings-api" /d:sonar.cs.opencover.reportsPath
 dotnet build BookingsAPI/BookingsApi.sln
 dotnet sonarscanner end
 
+##Branch name git hook will run on pre commit and control the standard for new branch name.
+
+The branch name should start with: feature/VIH-XXX  (X - is digit).
```

Comment:
And here also please change this message also to say the exact format that is needed. i.e. feature/VIH-XXX : Description